languages.yaml

```diff
@@ -138,6 +138,39 @@ Assembly:
       - yellow
       - green
     chip: "#6E4C13"
+Ats:
+  type: programming
+  ascii: |
+    {0}              ############
+    {0}          ####################
+    {0}       ##########################
+    {0}     ##############################
+    {0}   #####{1}LLLL{0}#####{2}atsatsatsatsatsatsa{0}#
+    {0}  ####{1}LLLLLLL{0}####{2}TSATSATSATSATSATSAT{0}##
+    {0} ####{1}LLL{0}##{1}LLL{0}############{2}SAT{0}###########
+    {0} ###{1}LLL{0}####{1}LLL{0}###{2}satsatsaTSAtsatsats{0}###
+    {0}###########{1}LLLL{0}##{2}ATSATSATSATSATSATSA{0}####
+    {0}##########{1}LLLLL{0}##{2}TS{0}######{2}ATS{0}######{2}AT{0}####
+    {0}#########{1}LLLLLL{0}##{2}SA{0}######{2}TSA{0}######{2}TS{0}####
+    {0} #######{1}LLL{0}##{1}LLL{0}#{2}ATsatsatSATsatsatSA{0}###
+    {0} ######{1}LLL{0}###{1}LLL{0}#{2}TSATSATSATSATSATSAT{0}###
+    {0}  ####{1}LLL{0}####{1}LLL{0}######################
+    {0}   ##{1}LLL{0}######{1}LLLLLLLLLLLLLLLLLLL{0}####
+    {0}     {1}LLL{0}#######{1}LLLLLLLLLLLLLLLLLL{0}##
+    {0}       ##########################
+    {0}          ####################
+    {0}              ############
+  colors:
+    ansi:
+      - yellow
+      - red
+      - blue
+    hex:
+      - "#FFFF00"
+      - "#FF0000"
+      - "#0000FF"
+    chip: "#1AC620"
+  serialization: ATS
```

Comment:
```suggestion
```
Our serialization is typically lowercase. This populates the available options for the `--ascii-language` option. You don't really need to use this option unless you're adding special characters (`CSharp` -> `c#`) or renaming (`Sh` -> `shell`).